bin/pdc_client

```diff
@@ -169,7 +169,7 @@ if __name__ == "__main__":
                 data[key] = ''
 
     try:
-        ssl_verify = options.ca_cert or not options.insecure
+        ssl_verify = options.ca_cert or options.insecure
```

Comment:
@simozhan ; If options.insecure is `None`, it will value `ssl_verify` -> `True`. As it to say, If there is no --insecure in client command line, we hope ssl_verify is None in line 176.